demo/infra/app/cross-vault-ns.tf

```diff
@@ -0,0 +1,263 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+# demonstrates how to setup cross Vault namespace for VSO and K8s auth when var.vault_enterprise = true
+
+# sets the group policy application to work with with cross Vault namespace auth
+resource "vault_generic_endpoint" "sys-group-policy-application" {
+  count = var.vault_enterprise ? 1 : 0
```

Comment:
The count on just this resource is a little strange, should this test only run against vault enterprise?
No change required, just wondering about its purpose?

Reply:
We only want to configure this policy if the demo is for Vault enterprise.